novawallet/Modules/AssetList/View/AssetListFlowLayout.swift

```diff
@@ -8,7 +8,7 @@ enum AssetListMeasurement {
     static let nftsHeight = 56.0
     static let assetHeight: CGFloat = 56.0
     static let assetHeaderHeight: CGFloat = 45.0
-    static let emptyStateCellHeight: CGFloat = 198
+    static let emptyStateCellHeight: CGFloat = 230
```

Comment:
Probably we should try to set the minimum height and then scale based on the context